broadway/read-model-mongodb/0.2/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "container": {
+        "mongodb_uri": "mongodb://127.0.0.1/",
+        "mongodb_database": "default"
```

Comment:
Same here. Consider using environment variables. 

Im not sure what happends when you reuse environment variables. Lets try =)